normalize-data.py

```diff
@@ -0,0 +1,74 @@
+import numpy
+import csv
+import functools
+
+def cleanF(n):
+  return float(n.strip().replace(",", ""))
+
+def readFile(url):
+  rawData=[]
+
+  with open(url) as f:
+    for l in csv.reader(f, quotechar='"', delimiter=',', quoting=csv.QUOTE_ALL, skipinitialspace=True):
+      rawData.append(l)
+
+  return rawData
+
+def writeFile(url, outputFile):
+  with open(url, 'w', encoding='UTF8') as outputFile:
+    writer = csv.writer(outputFile)
+    
+
+    writer.writerow(['date', 'high', 'low'])
+    for point in ouputData:
+      writer.writerow(point)
+
+def getDates(arr):
+  def flatten(x):
+      return x[0]
+
+  allDates=list(map(flatten, rawData))
+
+  return numpy.unique(allDates)
+
+def getHighsAndLows(date):
```

Comment:
iterates over the full data set, ignores irrelavant dates, picks high and low values from the set for that date